.github/workflows/ci_python.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/python_deps
 
-      - run: poetry run pylint src tests
+      - run: poetry run pylint src tests --rcfile pyproject.toml
 
```

Comment:
Can't get pylint to pick up the config in `pyproject.toml` without this...

Open to suggestions on an alternate fix.